src/api/classifier/classifier.py

```diff
@@ -1,6 +1,78 @@
-class ImageClassifier:
-    def __init__(self):
-        pass
+# -*- coding: utf-8 -*-
+"""Model_OOP.ipynb
 
-    def find(self, image):
-        return None
\ No newline at end of file
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1EltssuA0Fa_uX_ZRBbf51EtxqPmTt6XT
+"""
+
+#!pip install insightface
+#!pip install mxnet
+
+import insightface
+from PIL import Image
+from numpy import asarray, dot
+from matplotlib import pyplot as plt
+from numpy.linalg import norm
+import cv2
+
+model = insightface.app.FaceAnalysis(det_name='retinaface_mnet025_v2',rec_name='arcface_r100_v1', ga_name=None)
```

Comment:
should this be in the constructor (__init__)? 
some methods use the model attribute but it is out of the scope of the class

Reply:
what is the problem of this?!

Reply:
I think the attribute (model) should be encapsulated inside the class, not a global variable. 
this violates object-oriented design principles (encapsulation and abstraction)